crates/iroha_core/src/state.rs

```diff
@@ -65,8 +65,8 @@ use crate::{
 pub struct World {
     /// Iroha on-chain parameters.
     pub(crate) parameters: Cell<Parameters>,
-    /// Identifications of discovered trusted peers.
-    pub(crate) trusted_peers_ids: Cell<PeersIds>,
+    /// Identifications of discovered peers.
+    pub(crate) peers: Cell<Peers>,
```

Comment:
Yes, I think "trusted peers" here was actually a misnomer:

- trusted peers ... bootstrapping peers to establish the genesis network #1227
- peers in topology ... currently recognized peers on chain that can join consensus, regardless of connected or not
- connected peers in p2p ... peers that this peer actually has communication channels with